source/install/install-ubuntu-1804-mysql.rst

```diff
@@ -0,0 +1,57 @@
+.. _install-ubuntu-1804-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL.
+
+**To install MySQL on Ubuntu Server 18.04:**
+
+1. Log into the server that will host the database, and open a terminal window.
+
+2. Install MySQL.
+
+  ``sudo apt install mysql-server``
+
+3. Run the ``mysql_secure_installation`` and follow the steps
```

Comment:
Propose

```
Run ``mysql_secure_installation`` and follow the instructions. They are designed to set a few initial parameters that are more secure than the default Ubuntu (and MySQL) installation.
```

Reply:
^FYI - I updated the text. I had a second sentence included, but it's not necessary.

Reply:
done @jasonblais PTAL

Reply:
@cpanato I see the original text still, perhaps you forgot to push?

Reply:
yep, I pushed to origin with is wrong :(
should be fixed now